exercises/practice/anagram/test_anagram.R

```diff
@@ -10,25 +10,26 @@ test_that("no matches", {
                c())
 })
 
-test_that("detects simple anagram", {
```

Comment:
There is no distinction between simple and complex anagrams. Removing this adjective.